docs/community/developer_guide.mdx

```diff
@@ -89,18 +89,20 @@ This section needs a lot of love and automation 🙂
 1. Open a bump PR to build with the latest golang: replace go version of the [binary](https://github.com/oras-project/oras/blob/main/.github/workflows/release-github.yml#L32) and [image](https://github.com/oras-project/oras/blob/main/Dockerfile#L14) to match the latest stable version. Skippable if the latest version is already used.
 1. (patch-release-only) Cherry-pick or backport the fix commits to the patch release branch. Make sure applicable critical and high CVE patches are on the release branch.
 
-### Request to vote
+### Request to vote and merge
 1. Open a PR to update the `oras` version: replace the [current version](https://github.com/oras-project/oras/blob/main/internal/version/version.go#L20) with the upcoming release version.
     - The target branch is:
         - `main` if releasing a new version.
         - `release-<major>.<minor>` if releasing a patch.
     - The title should be `bump: tag and release ORAS CLI v<major>.<minor>.<patch>[-<pre-release>]`.
     - The description must reference the digest of version bump up commit for voting.
-1. Send a message to the ORAS slack channel to call for a vote on the release PR. If the vote PR has a super-majority of approval from ORAS maintainers, then the PR could be merged into the target branch.
+1. Send a message to the ORAS slack channel to call for a vote on the release PR.
```

Comment:
```suggestion
1. Send a message to the ORAS slack channel to call for a vote on the release PR. If the vote PR has a super-majority of approval from ORAS maintainers, then the PR could be merged into the target branch.
```

Reply:
Suggest keeping this criteria because this is a part of governance process. If we feel super-majority is vague, we can say "super-majority (Two-thirds)" here. Maintainers are responsible for the release and its quality.

Reply:
This criteria is not removed but moved to next step.

Reply:
Send a vote and merging are two separate operations, it's better to split them into two steps.

Reply:
Oh, sorry for missing that line below. This looks good me!